packages/runtime/container-runtime/src/deltaManagerProxies.ts

```diff
@@ -208,10 +208,16 @@ export class DeltaManagerSummarizerProxy extends BaseDeltaManagerProxy {
 export class DeltaManagerPendingOpsProxy extends BaseDeltaManagerProxy {
 	public get minimumSequenceNumber(): number {
 		const minPendingSeqNum = this.pendingStateManager.minimumPendingMessageSequenceNumber;
-		// There is a chance that minPendingSeqNum is greater than minimum sequence number.
-		// minPendingSeqNum is based on the pending ops, so it's based on ref seq number.
-		// Imagine an op has just be sent while there's another client that has been lagging behind,
-		// it will likely have a ref seq number greater than the minimum seq number.
+		/**
+		 * The reason why the minimum pending sequence number can be less than the delta manager's minimum sequence number (DM's msn)
+		 * is that when we are processing messages in the container runtime/delta manager, the delta manager's minimum
+		 * sequence number can be updated to continually increase. In the meantime, the pending state manager's op which
+		 * hasn't been sent can still have a lower sequence number than the DM's msn (think about a disconnected scenario). To successfully
+		 * rebase that pending op to have a reference sequence number greater than or equal to the DM's msn, we need to resubmit that op back to the
+		 * DDS. The DDS still needs to keep the local data for that op that has a reference sequence number lower than the
```

Comment:
I would remove this part because it is covered in the sentence - "The DDS still needs to keep the local data for that op that has a reference sequence number lower than the DM's msn".
Or you could add that it could be lower than DM's msn but higher than min pending seq#.

Reply:
Added that it could be lower than DM's msn but higher than min pending seq#.